guides/v2.2/cloud/project/project-conf-files_services-rabbit.md

```diff
@@ -10,31 +10,38 @@ The [Message Queue Framework (MQF)]({{ page.baseurl }}/config-guide/mq/rabbitmq-
 
 The MQF uses [RabbitMQ](http://www.rabbitmq.com) as the messaging broker, which provides a scalable platform for sending and receiving messages. It also includes a mechanism for storing undelivered messages. RabbitMQ is based on the Advanced Message Queuing Protocol (AMQP) 0.9.1 specification.
 
-We support RabbitMQ version 3.5.
-
 {:.bs-callout .bs-callout-warning}
 If you prefer using an existing AMQP-based service, like RabbitMQ, instead of relying on {{site.data.var.ece}} to create it for you, use the [`QUEUE_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#queue_configuration) environment variable to connect it to your site.
 
 {% include cloud/service-config-integration-starter.md %}
 
-## Add RabbitMQ in services.yaml and .magento.app.yaml {#settings}
+{:.procedure}
+To enable RabbitMQ:
 
-To enable RabbitMQ, add the following code with your installed version and allocated disk space in MB to the `.magento/services.yaml` file.
+1. Add the required name, type, and version to the `.magento/services.yaml` file along with the the installed RabbitMQ version and allocated disk space in MB:
```

Comment:
Technically, version is not a service value as defined in the main services topic. It is more a value of a value.

```suggestion
1. Add the required name, type, and disk value (in MB) to the `.magento/services.yaml` file along with the the installed RabbitMQ version.
```